spec/components/check_answers_summary_list_card_component_spec.rb

```diff
@@ -5,103 +5,68 @@
 
   let(:rendered) { render_inline(component) }
 
-  context "when before 23/24 collection" do
-    before do
-      Timecop.freeze(Time.zone.local(2023, 1, 10))
-      Singleton.__init__(FormHandler)
-    end
-
-    after do
-      Timecop.return
-    end
-
-    context "when given a set of questions" do
-      let(:user) { build(:user) }
-      let(:log) { build(:lettings_log, :completed, age2: 99, startdate: Time.zone.local(2021, 5, 1)) }
-      let(:subsection_id) { "household_characteristics" }
-      let(:subsection) { log.form.get_subsection(subsection_id) }
-      let(:questions) { subsection.applicable_questions(log) }
-
-      it "renders a summary list card without question numbers for the answers to those questions" do
-        expect(rendered).to have_content(questions.first.answer_label(log))
-        expect(rendered).to have_content("Lead tenant’s age")
-        expect(rendered).not_to have_content("Q32 - Lead tenant’s age")
-      end
-
-      it "applicable questions doesn't return questions that are hidden in check answers" do
-        expect(component.applicable_questions.map(&:id).include?("retirement_value_check")).to eq(false)
-      end
-
-      it "has the correct answer label for a question" do
-        sex1_question = questions[2]
-        expect(component.get_answer_label(sex1_question)).to eq("Female")
-      end
-
-      context "when log was created via a bulk upload and has an unanswered question" do
-        subject(:component) { described_class.new(questions:, log:, user:) }
-
-        let(:bulk_upload) { create(:bulk_upload) }
-        let(:log) { build(:lettings_log, :in_progress, creation_method: "bulk upload", age2: 99, startdate: Time.zone.local(2021, 5, 1), bulk_upload:) }
-
-        it "displays tweaked copy in red" do
-          expect(rendered).to have_selector("span", class: "app-!-colour-red", text: "You still need to answer this question")
-        end
-      end
+  let(:user) { create(:user) }
+  let(:log) { create(:lettings_log, :completed, age2: 99, retirement_value_check: 1) }
```

Comment:
if we can get away with building the log below, can we get away with it here?

Reply:
I think it needs creating, not sure why, but without it questions ends up being nil